### ISSUE TITLE
update to 2.0.4

### ISSUE THREAD

**Issue by JustinBenedick** [LacledesLAN/gamesvr-tf2classic#3]
Need to update server to https://tf2classic.com/?id=5610   or figure out how to use auto updater.   https://wiki.tf2classic.com/wiki/Updating

**Comment by dudleycodes** [comment]
Their self-installer does a full update each time - it can't download just updates. And it's crashing for me about 1/3 of the times. And it has prompts that make it difficult to use in a Docker image. So, want to avoid it at all costs.

After updating to 2.0.4, getting this error:

```shell
RUNNING TEST: gamesvr-tf2classic-160313
Command: /app/srcds_run -game tf2classic +map ctf_2fort -insecure -norestart +sv_lan 1
Running under uid=999(TF2Classic) gid=0(root) groups=0(root)

0...5...terminated.

Boot error: Test process self-terminated

[LOGFILE OUTPUT]
»»  Auto detecting CPU
»»  Using default binary: ./srcds_linux
»»  Using Breakpad minidump system. Version: 5394426 AppID: 244310
»»  Setting breakpad minidump AppID = 244310
»»  Using breakpad crash handler
»»  Loaded 8 VPK file hashes from /app/tf2classic/custom/4team_live_buildables.vpk for pure server operation.
»»  Loaded 8 VPK file hashes from /app/tf2classic/custom/4team_live_buildables.vpk for pure server operation.
»»  Loaded 8 VPK file hashes from /app/tf2classic/custom/4team_live_buildables.vpk for pure server operation.
»»  Loaded 6413 VPK file hashes from /app/tf2classic/vpks/tf2c_textures.vpk for pure server operation.
»»  Loaded 6413 VPK file hashes from /app/tf2classic/vpks/tf2c_textures.vpk for pure server operation.
»»  Loaded 545 VPK file hashes from /app/tf2classic/vpks/tf2c_sound_vo_english.vpk for pure server operation.
»»  Loaded 545 VPK file hashes from /app/tf2classic/vpks/tf2c_sound_vo_english.vpk for pure server operation.
»»  Loaded 814 VPK file hashes from /app/tf2classic/vpks/tf2c_sound_misc.vpk for pure server operation.
»»  Loaded 814 VPK file hashes from /app/tf2classic/vpks/tf2c_sound_misc.vpk for pure server operation.
»»  Loaded 1103 VPK file hashes from /app/tf2classic/vpks/tf2c_misc.vpk for pure server operation.
»»  Loaded 1103 VPK file hashes from /app/tf2classic/vpks/tf2c_misc.vpk for pure server operation.
»»  Loaded 1103 VPK file hashes from /app/tf2classic/vpks/tf2c_misc.vpk for pure server operation.
»»  Loaded 1222 VPK file hashes from /app/hl2/hl2_textures.vpk for pure server operation.
»»  Loaded 574 VPK file hashes from /app/hl2/hl2_sound_vo_english.vpk for pure server operation.
»»  Loaded 383 VPK file hashes from /app/hl2/hl2_sound_misc.vpk for pure server operation.
»»  Loaded 446 VPK file hashes from /app/hl2/hl2_misc.vpk for pure server operation.
»»  Loaded 446 VPK file hashes from /app/hl2/hl2_misc.vpk for pure server operation.
»»  Loaded 5 VPK file hashes from /app/platform/platform_misc.vpk for pure server operation.
»»  Loaded 5 VPK file hashes from /app/platform/platform_misc.vpk for pure server operation.
»»  server_srv.so loaded for "Team Fortress 2 Classic"
»»  memy::GetModInfo -> Couldn't find info for modname server.so
»»  memy::InitSingleBin -> GetModuleInformation failed for server.so!
»»  Missing shutdown function for sv.Init( bDedicated ) : sv.Shutdown()
»»  Missing shutdown function for g_GameEventManager.Init() : g_GameEventManager.Shutdown()
»»  Missing shutdown function for NET_Init( bDedicated ) : NET_Shutdown()
»»  Missing shutdown function for Filter_Init() : Filter_Shutdown()
»»  Missing shutdown function for COM_Init() : COM_Shutdown()
»»  Missing shutdown function for g_pCVar->Init() : g_pCVar->Shutdown()
»»  Missing shutdown function for Cmd_Init() : Cmd_Shutdown()
»»  Missing shutdown function for Cbuf_Init() : Cbuf_Shutdown()
»»  Missing shutdown function for Con_Init() : Con_Shutdown()
»»  Missing shutdown function for Memory_Init() : Memory_Shutdown()
»»  Missing shutdown function for Host_Init( s_bIsDedicated ) : Host_Shutdown()
»»  Missing shutdown function for Sys_InitMemory() : Sys_ShutdownMemory()
»»  Missing shutdown function for Sys_Init() : Sys_Shutdown()
»»  Missing shutdown function for COM_InitFilesystem( info.m_pInitialMod ) : COM_ShutdownFileSystem()
»»  pure virtual method called
»»  terminate called without an active exception
»»  Aborted
```

**Comment by dudleycodes** [comment]
The fix for this `Couldn't find info for modname server.so` issue appears to be:

```shell
TF2Classic@b3b8a374634f:~$ rm -rf /app/tf2classic//bin/server_srv.so
TF2Classic@b3b8a374634f:~$ ln -s /app/tf2classic/bin/server.so /app/tf2classic/bin/server_srv.so
```

For reference: [updating](https://wiki.tf2classic.com/wiki/Updating) ([archive](https://archive.ph/0b39I)):

> you should make sure that the "server_srv.so" file in your tf2classic/bin folder is a symlink to server.so in the same folder. It cannot be a copy of the file.

**Comment by dudleycodes** [comment]
Fixed with commit 0d8404b6cc7af6afac676fff6d655ad24d0f24b6
